Applications/MainApplication/Gui/MainWindow.cpp

```diff
@@ -316,6 +321,10 @@ void MainWindow::onSelectionChanged( const QItemSelection& selected,
             } else
             { m_currentShaderBox->setCurrentText( shaderName.c_str() ); }
         }
+        if ( dynamic_cast<Engine::Camera*>( ent.m_component ) )
+        {
+            m_useCameraButton->setEnabled( true );
+        }

```

Comment:
Why using a dynamic cast to test if the component is a camera ? Prefer to add a isCameraNode method to do that.

Reply:
The thing is I couldn't find another way to check that